stripe/resource.py

```diff
@@ -759,23 +786,22 @@ class Customer(CreateableAPIResource, UpdateableAPIResource,
 
     def add_invoice_item(self, idempotency_key=None, **params):
         params['customer'] = self.id
-        ii = InvoiceItem.create(self.api_key,
-                                idempotency_key=idempotency_key, **params)
+        ii = InvoiceItem.create(idempotency_key=idempotency_key, **params)
         return ii
 
     def invoices(self, **params):
         params['customer'] = self.id
-        invoices = Invoice.list(self.api_key, **params)
+        invoices = Invoice.list(**params)
```

Comment:
Doesn't this change/break the old behavior. It seems like before we were using the API key set on the customer and now we're just using one that gets passed in with `params`.